src/Status/Api.elm

```diff
@@ -1,22 +1,22 @@
 module Status.Api exposing (getStatus)
 
-import Utils.Api exposing (baseUrl)
-import Http
-import Status.Types exposing (StatusMsg(NewStatus), StatusResponse)
+import Utils.Api exposing (baseUrl, send, get)
+import Utils.Types exposing (ApiData)
+import Status.Types exposing (StatusResponse)
 import Json.Decode exposing (Decoder, map2, string, field, at)
 import Types exposing (Msg(MsgForStatus))
 
 
-getStatus : Cmd Msg
-getStatus =
+getStatus : (ApiData StatusResponse -> Msg) -> Cmd Msg
```

Comment:
Same here `(ApiData StatusResponse -> Msg) -> Cmd Msg` can be written as `(ApiData StatusResponse -> msg) -> Cmd msg` so that `import Types exposing (Msg(MsgForStatus))` may removed.